packages/examples/src/example.ts

```diff
@@ -6,5 +6,7 @@ export interface ExampleDescription {
   data: any;
   schema: JsonSchema;
   uiSchema: UISchemaElement;
+  translations?: any;
+  locale?: String;
```

Comment:
you could assign a default value to locale, so you don't have wo handle this in the changeExample method